oak_abi/proto/application.proto

```diff
@@ -153,6 +154,20 @@ message HttpClientConfiguration {
   // to set the privilege of the node.
   string authority = 1;
 }
+// HttpCredentialedClientConfiguration describes the configuration of an HTTP client
+// pseudo-Node (which is provided by the Oak Runtime) that uses a client certificate to connect to
+// the remote server, connected to a specific
+// external (non-Oak) HTTP or HTTPS service.
```

Comment:
You can run `./scripts/runner format` for auto-formatting
I think the CI is failing because of that